sandbox/apparmor/apparmor.go

```diff
@@ -32,7 +32,9 @@ import (
 
 	"github.com/snapcore/snapd/dirs"
 	"github.com/snapcore/snapd/osutil"
+	"github.com/snapcore/snapd/snapdtool"
 	"github.com/snapcore/snapd/strutil"
+	"github.com/snapcore/snapd/testutil"
```

Comment:
we can't use testutil in the prod code itself, it brings in go-check

Reply:
Yep mardy mentioned this too - I moved this to export_test in 76a8d13428